src/common/spawn.cpp

```diff
@@ -101,6 +102,7 @@ int spawn(const char *filename, const std::vector<std::string>& args, bool wait)
   // child
   if (pid == 0)
   {
+    tools::closefrom(3);
```

Comment:
Should it close all the descriptors here? I'm not sure there is a good generic behavior here; just wanted to discuss it.

Reply:
Typical behaviour I've seen before is to keep stdin/out/err alone. I suppose I could hook onto stdout/err to sent to the logging system though...

Reply:
Also works with 0 closed. I think it's safer. Keeping 1 and 2 open is probably best so any errors can be seen.